recipes/radis/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "radis" %}
+{% set version = "0.13.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 650ef57b6f6b96232e4282564b613650538ff914268e71b67a4ea846c9a8d746
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy <=1.22.3
+    
+    
+  run:
+    - python
+    - astropy # Unit aware calculations
+    - astroquery >=0.3.9 # to fetch HITRAN databases
+    - beautifulsoup4 # parse ExoMol website
+    - configparser
+    - cython
+    - h5py
+    - habanero # CrossRef API to retrieve data from doi
+    - ipython >=7.0.0
+    - joblib  # for parallel loading of SpecDatabase
+    - lxml  # parser used for ExoMol website
+    - matplotlib-base
+    - mpldatacursor
+    - numba # for line survey HTML output
+    - pandas >=1.0.5
+    - peakutils
+    - plotly >=2.5.1 # for line survey HTML output
+    - progressbar2 # used in vaex
+    - psutil # for getting user RAM
+    - pytest #testing
+    - scipy >=1.4.0
+    - seaborn # other matplotlib themes
+    - termcolor # terminal colors
+    - vaex >=4.9.2 # load HDF5 files
+    - hjson-py # Json with comments (for default_radis.json) 
+    - pytables # for pandas to HDF5 export
+    - json_tricks # to deal with non jsonable formats
```

Comment:
All the comments here seems to belong in an upstream documentation. They are not very helpful here in the package.